pkg/api/event/nonblocking/user_force_update_password_changed.go

```diff
@@ -0,0 +1,49 @@
+package nonblocking
+
+import (
+	"github.com/authgear/authgear-server/pkg/api/event"
+	"github.com/authgear/authgear-server/pkg/api/model"
+)
+
+const (
```

Comment:
Change the list of the new events to

```
password.primary.reset
password.primary.changed
password.secondary.changed
password.primary.force_changed
password.secondary.force_changed
```

Document these new events in specs/event.md

Reply:
Question - do we support force-change 2ndary password?